x-pack/solutions/security/plugins/security_solution/server/endpoint/routes/resolver/entity/utils/supported_schemas.test.ts

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getSupportedSchemas } from './supported_schemas';
+import type { ExperimentalFeatures } from '../../../../../../common';
+import * as securityModules from './security_modules';
+
+jest.mock('./security_modules', () => ({
+  CORE_SECURITY_MODULES: [
+    'crowdstrike',
+    'jamf_protect',
+    'sentinel_one',
+    'sentinel_one_cloud_funnel',
+  ],
+  MICROSOFT_DEFENDER_MODULES: ['microsoft_defender_endpoint', 'm365_defender'],
+  getSecurityModuleDatasets: jest.fn(),
+  getAllSecurityModules: jest.fn(),
+}));
```

Comment:
Since we're mocking the entire module, do we need to duplicate these constant arrays in the mock? Could we import the real constants and just mock the functions we need to control?